src/shipit_code_coverage/tests/conftest.py

```diff
@@ -243,13 +243,3 @@ def tobytes(x):
                     os.path.join(local, '.hg/pushlog2.db'))
 
     return local
-
-
-@pytest.fixture()
-def ACTIVEDATA_CHUNK_TO_TESTS():
```

Comment:
Don't you need to mock more active data results instead of removing it ?

Reply:
I've decided to create "fake" mocks as it's easier to test, the mocks generated by actually querying activedata would have been huge.